### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by imollov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,23 +41,24 @@ Start up the local Hardhat network and deploy all contracts
...
 ## Environment Variables
 
 Deploying to a public network requires setting `RPC_URL` [environment variables](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa). If you're not running a node you can get one at [Infura's site](https://infura.io).
 
 Additionally you'll need to set a `MNEMONIC` variable from your wallet, ie Metamask. The account must have enough funds to deploy the contracts, as well as LINK which can be obtained from faucets. It is also possible to set a `PRIVATE_KEY` instead with some changes in `hardhat.config.js`.
 
-Running the front-end app in read-only mode also requires setting an environment variable `INFURA_KEY`.
+The front-end app also requires setting an environment variable `INFURA_KEY` to be able to run in read-only mode and use WalletConnect.
 
 To make this easier there are `.env` example files in the `hardhat` and `frontend` workspaces.
```
Comment (mradkov) [inline README.md:64]: Can we add a table here with all of the required env variables?

Comment (imollov) [reply README.md:64]: Will try it in an upcoming PR about improving the readme.

File: packages/frontend/components/feeds/SelectFeed.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function SelectFeed(): JSX.Element {
 
   return (
     <>
-      {chainId !== ChainId.Kovan && (
+      {chainId === ChainId.Rinkeby && (
```
Comment (mradkov) [inline packages/frontend/components/feeds/SelectFeed.tsx:22]: Does this mean the feeds only work on Rinkeby?

Comment (imollov) [reply packages/frontend/components/feeds/SelectFeed.tsx:22]: It doesn't work only on Rinkeby, from the set of supported networks - Kovan, Rinkeby, Localhost.

Comment (mradkov) [reply packages/frontend/components/feeds/SelectFeed.tsx:22]: Oops my bad, overlooked. Thanks!

**Review by mradkov — APPROVED** [review]

**Pull request merged**